projects/backend/controllers/issue.ts

```diff
@@ -33,11 +34,21 @@ export const issueController = (req: Request, res: Response) => {
         .catch(e => console.error(e))
 }
 
+const getEditionOrFallback = (editionType: string) =>
+    editionType ? editionType : 'daily-edition'
```

Comment:
This is equivalent to `editionType || 'daily-edition'` IIUC

Reply:
done